test/e2e_node/e2e_service.go

```diff
@@ -314,9 +314,15 @@ type killCmd struct {
 }
 
 func (k *killCmd) Kill() error {
+	if k == nil {
+		glog.V(2).Infof("The killCmd is nil, nothing will be killed")
```

Comment:
Like, you could also set the override command to `echo "no need to kill"` in that case...